gslib/utils/text_util.py

```diff
@@ -62,22 +78,13 @@ def AddQueryParamToUrl(url_str, param_name, param_value):
     (str or unicode) A string representing the modified url, of type `unicode`
     if the url_str argument was a `unicode`, otherwise a `str` encoded in UTF-8.
   """
-  url_was_unicode = isinstance(url_str, unicode)
-  if isinstance(url_str, unicode):
-    url_str = url_str.encode('utf-8')
-  if isinstance(param_name, unicode):
-    param_name = param_name.encode('utf-8')
-  if isinstance(param_value, unicode):
-    param_value = param_value.encode('utf-8')
-  scheme, netloc, path, query_str, fragment = urlparse.urlsplit(url_str)
-
-  query_params = urlparse.parse_qsl(query_str, keep_blank_values=True)
+  scheme, netloc, path, query_str, fragment = urllib.parse.urlsplit(url_str)
```

Comment:
Do we expact all arguments to be unicode now?  I'm curious if the notes in the docstring about types still apply or not.